src/killEvent.js

```diff
@@ -1,17 +1,14 @@
 function killEvent(target, name, descriptor) {
-  let oldHandler = target;
 
-  function eventCanceler(...args) {
-    args[0].stopPropagation();
-    args[0].preventDefault();
+  function eventCanceler(event) {
+    event.stopPropagation();
+    event.preventDefault();
 
-    oldHandler.apply(this, args);
+    return target.apply(this, arguments);
```

Comment:
I agree.
Just for the record, we are following the airbnb style guide: https://github.com/airbnb/javascript#7.6 that advises against using "arguments".
Your changes do not go against what airbnb was talking about and I get that. Thanks